contracts/modules/commons/ModuleHooks.sol

```diff
@@ -1,32 +1,63 @@
 pragma solidity ^0.6.6;
 pragma experimental ABIEncoderV2;
 
-import "./ModuleBase.sol";
+import "./ModuleSelfAuth.sol";
+import "./ModuleStorage.sol";
 
 import "../../interfaces/receivers/IERC1155Receiver.sol";
 import "../../interfaces/receivers/IERC721Receiver.sol";
 
 
-contract ModuleHooks is ModuleBase, IERC1155Receiver, IERC721Receiver {
-  mapping(bytes4 => address) public hooks;
+contract ModuleHooks is ModuleSelfAuth, IERC1155Receiver, IERC721Receiver {
+  //                       HOOKS_KEY = keccak256("org.arcadeum.module.hooks.hooks");
```

Comment:
Should add a test that these keys are the expected hashes, in-case we change the name without updating the comment or we change the key hash by accident

Reply:
Good idea!